clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -161,6 +161,7 @@ class ClioClient(webClient: ClioWebClient,
     ClioCommand.help
       .messagesMap(command)
       .helpMessage(s"${beforeCommandHelp.progName} [options]", command)
+      .trim()
```

Comment:
Does this undo the doubled newlines above?

Reply:
Other way around, but either way they balance out.

The underlying problem I'm trying to fix is that the spacing between command help-blocks is inconsistent: commands with options get 1 trailing line of whitespace, and those without get 2 trailing lines. It's a bug in case-app's logic for generating help blocks.

I'm working around it by stripping all trailing whitespace from the text generated by case-app, then adding the newlines, so every command help-block is followed by a blank line.